qaframework-bdd-tests/docker/docker-compose-refqa-3x.yml

```diff
@@ -0,0 +1,57 @@
+version: "3.7"
+
+services:
+  # Proxy
+  proxy:
+    image: openmrs/openmrs-reference-application-3-gateway:latest
+    depends_on:
+      - frontend
+      - openmrs
```

Comment:
Wouldn't it be better to rename this from `openmrs` to `backend`?

Reply:
That would be nice! <3

Reply:
So, the proxy's configuration unfortunately [hard-codes](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/6fa42a5dd8071b2726368bba91d854dc15e9a91b/run/docker/proxy/nginx.conf#L39) the name `openmrs`. I think you might be able to get away with:

```yml
  proxy:
    image: openmrs/openmrs-reference-application-3-gateway:latest
    depends_on:
      - frontend
      - backend
    links:
      - backend:openmrs
    ports:
      - "80:80"

# ...

  backend:
    image: openmrs/openmrs-reference-application-3-backend:latest
    depends_on:
      - db
    environment:
      OMRS_CONFIG_MODULE_WEB_ADMIN: "true"
      OMRS_CONFIG_AUTO_UPDATE_DATABASE: "true"
      OMRS_CONFIG_CREATE_TABLES: "true"
      OMRS_CONFIG_CONNECTION_SERVER: db
      OMRS_CONFIG_CONNECTION_DATABASE: openmrs
      OMRS_CONFIG_CONNECTION_USERNAME: openmrs
      OMRS_CONFIG_CONNECTION_PASSWORD: Admin123
    healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:8080/openmrs"]
      timeout: 5s
    volumes:
      - openmrs-data:/openmrs/data
```

Or a custom nginx.conf file [like we do for dev3.openmrs.org](https://github.com/openmrs/openmrs-contrib-ansible-docker-compose/tree/master/files/emr-3-dev).

Of course, I suppose we could just change the image too...

Reply:
I don't think it's worth adding any complexity for this.